code/modules/cargo/exports/xenobio.dm

```diff
@@ -0,0 +1,43 @@
+/datum/export/slime
+
+/datum/export/slime/grey
+	cost = 150
+	unit_name = "grey slime core"
+	export_types = list(/obj/item/slime_extract/grey)
+
+/datum/export/slime/common
+	cost = 200
+	unit_name = "common slime core"
+	export_types = list(/obj/item/slime_extract/metal,/obj/item/slime_extract/orange,/obj/item/slime_extract/purple,/obj/item/slime_extract/blue)
+
+/datum/export/slime/uncommon
+	cost = 400
+	unit_name = "uncommon slime core"
+	export_types = list(/obj/item/slime_extract/gold,/obj/item/slime_extract/green,/obj/item/slime_extract/red,/obj/item/slime_extract/pink)
+
+/datum/export/slime/rare
+	cost = 600
+	unit_name = "rare slime core"
+	export_types = list(/obj/item/slime_extract/silver,/obj/item/slime_extract/darkblue,/obj/item/slime_extract/darkpurple,/obj/item/slime_extract/yellow)
+
+/datum/export/slime/charged
+	cost = 1000
+	unit_name = "charged slime core"
+	export_types = list(/obj/item/stock_parts/cell/high/slime)
+
+/datum/export/slime/hypercharged
```

Comment:
couldn't you just sell the 10 yellow slime cores (plus however many yellow slime cores are in a crossbreeding-ready yellow slime) needed to make this for 6,000 credits?